hack/devtools/local_dev_env.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Local development environment script.
+# Execute this script from the root folder of the repo (ARO-RP).
+# This script is aimed to provide an automatic and easy way to prepare 
+# the environment and execute the ARO RP locally.
+# The steps here are the ones defined in docs/deploy-development-rp.md
+# We recommend to use this script after you understand the steps of the process, not before.
+
+
+build_development_az_aro_extension() {
+    echo "INFO: Building development az aro extension..."
+    make az
+}
+
+verify_aro_extension() {
+    echo "INFO: Verifying aro extension..."
+    grep -q 'dev_sources' ~/.azure/config || cat >>~/.azure/config <<EOF
+[extension]
+dev_sources = $PWD/python
+EOF
+}
+
+set_storage_account() {
+    echo "INFO: Setting storage account..."
+    export SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets
+}
+
+apply_default_env_config() {
+    echo "INFO: Applying default environment config file..."
+    cat >env <<EOF
+export LOCATION=eastus
+export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
+export RP_MODE=development # to use a development RP running at https://localhost:8443/
+
+source secrets/env
+EOF
+    source_env
+}
+
+ask_to_create_Azure_deployment() {
+    local answer
+    read -p "Create Azure deployment in the current subscription ($AZURE_SUBSCRIPTION_ID)? (y / n / l (list existing deployments)) " answer
+
+    if [[ "$answer" == "y" ]]; then
+        create_Azure_deployment
+    elif [[ "$answer" == "n" ]]; then
+        echo "Skipping creation of Azure deployment..."
+    elif [[ "$answer" == "l" ]]; then
+        list_Azure_deployments_and_ask_to_create_one
```

Comment:
You could have one less function by doing this: 

```suggestion
        list_Azure_deployment_names
        ask_to_create_Azure_deployment
```
and removing list_Azure_deployments_and_ask_to_create_one()

Reply:
Yes, thanks! Forgot to remove the duplication after copy/paste. Done!